### ISSUE TITLE
something for C

### ISSUE THREAD

**Issue by Mis012** [dalerank/nngui#2]
Hello, considering this is not going to work with C(99/11/ANSI), do you know of any port of this that does, or some C nanovg GUI toolkit? Thanks.

**Comment by dalerank** [comment]
No, but you may port it self, it easy, I did it for my last work, but havnt
source also

17 мар. 2018 г. 15:50 пользователь "Mis012" <notifications@github.com>
написал:

**Comment by Mis012** [comment]
Even if I don't know C++?

**Comment by dalerank** [comment]
Hm... it a reason:(

17 мар. 2018 г. 16:27 пользователь "Mis012" <notifications@github.com>
написал:
